Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.xaml

```diff
@@ -472,39 +585,32 @@
                             </ContentControl>
 
                             <Grid Grid.Row="1" 
-                                      Grid.RowSpan="1" 
-                                      Grid.Column="0" 
-                                      Grid.ColumnSpan="2"
-                                      x:Name="PART_MainContent"
-                                      Visibility="Collapsed"
-                                      extensions:VisualEx.NormalizedCenterPoint="0"
-                                      Background="{TemplateBinding Background}">
-
-                                <ContentPresenter HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
-                                                      HorizontalContentAlignment="Stretch"
-                                                      VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
-                                                      VerticalContentAlignment="Stretch" >
-                                    <animations:Implicit.ShowAnimations>
-                                        <animations:OpacityAnimation Duration="0:0:0.3" From="0" To="1" Delay="0:0:0.25" SetInitialValueBeforeDelay="True"></animations:OpacityAnimation>
-                                        <animations:TranslationAnimation Duration="0:0:0.3" From="0, 20, 0" To="0" Delay="0:0:0.25"></animations:TranslationAnimation>
-                                    </animations:Implicit.ShowAnimations>
-
-                                    <animations:Implicit.HideAnimations>
-                                        <animations:OpacityAnimation Duration="0:0:0.2" To="0"></animations:OpacityAnimation>
-                                    </animations:Implicit.HideAnimations>
+                                  Grid.RowSpan="1" 
+                                  Grid.Column="0" 
+                                  Grid.ColumnSpan="2"
+                                  x:Name="PART_MainContent"
+                                  Visibility="Collapsed"
```

Comment:
I think the reason why the collapsing animations don't run is because the Visibility is always being reset to Collapsed. You can just remove this line and it should work